app/javascript/groceries/components/Item.vue

```diff
@@ -30,11 +30,13 @@
         edit-icon(size='17')
     | &nbsp;
     span ({{item.needed}})
-  .delete.text-2xl.px-2.js-link.right.text-red-600.leading-unset(
+  .delete.text-2xl.js-link.right.text-red-600.leading-unset(
     v-if='ownStore'
-    @click="groceriesStore.destroyItem({ item })"
-    title='Delete item'
-  ) ×
+  )
+    button.px-2(
```

Comment:
(Needed to make this into a proper button so that the feature spec could click on it (which also makes clear the accessibility need for this to be a proper button).)